src/components/DroppableList.jsx

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import { Flex, Grid, GridItem, Heading, VStack } from '@chakra-ui/react';
 import DraggableItem from './DraggableItem';
 
-const DroppableList = ({ name, species, colID, searchItem }) => {
+const DroppableList = ({ name, species, colID, searchItem, editSpecies }) => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'name' is defined but never used.